lib/ShareByCircleProviderDeprecated.php

```diff
@@ -378,10 +378,11 @@ private function createShareChild($userId, $share) {
 	 * @param Folder $node
 	 * @param bool $reshares Also get the shares where $user is the owner instead of just the
 	 *     shares where $user is the initiator
+	 * @param bool $shallow Whether the method should stop at the first level, or look into sub-folders.
 	 *
 	 * @return Share[]
 	 */
-	public function getSharesInFolder($userId, Folder $node, $reshares) {
+	public function getSharesInFolder($userId, Folder $node, $reshares, $shallow = true) {
```

Comment:
et bool !

Reply:
Incompatible with the interface if I add bool, so I am leaving it like this, sorry :/